### ISSUE TITLE
Gradle plugin for result aggregation: camunda-process-test-coverage-report-aggregator-gradle-plugin

### ISSUE THREAD

**Issue by JZoricGenerali** [camunda-community-hub/camunda-process-test-coverage#265]
It will be nice to create the gradle plug-in for result aggregation:
https://camunda-community-hub.github.io/camunda-process-test-coverage/snapshot/getting-started.html#aggregating-results

**Comment by TheTankMan** [comment]
I agree, having the result aggregation for Process test will usefull for reporting purposes and also for checking what has and what has not been tested

**Comment by iamvinit** [comment]
Any plan to release this plugin?

**Comment by iamvinit** [comment]
Any plan to release this plugin?

**Comment by rohwerj** [comment]
Wanted to wait for https://github.com/camunda-community-hub/camunda-process-test-coverage/issues/259 to be resolved. But it seems it will take a while longer, so I'll do a release of 2.1.0 in the next days.
